File: mau/58278.pas
function ucln(a, b : integer) : integer;
var r : integer;
begin
    while b<>0 do begin
        r:= a mod b;
        a := b;
        b:= r;
    end;
    exit(a)
end;
var x,y,z,a,b,c,m,n,d : integer;
BEGIN
    readln(x, a);
    readln(y, b);
    readln(z, c);
    m := x*b*c + y*a*c - z*a*b;
    n := a*b*c;
    d := ucln(m, n);
    write(m div d,' ', n div d);
END.

File: mau/58278.py
def ucln(a, b):
    while b != 0:
        r = a % b;
        a = b
        b = r
    return a
x, a = [int(x) for x in input().split()]
y, b = [int(x) for x in input().split()]
z, c = [int(x) for x in input().split()]
m = x*b*c + y*a*c - z*a*b
n = a*b*c
d = ucln(m, n)
print(m//d, n // d)
